ffxiv_undercut.py

```diff
@@ -48,13 +103,16 @@ def create_undercut_message(json_response, webhook_url):
     auctions_by_retainer = organize_by_retainer(json_response.get("auction_data", {}))
 
     for retainer, details in auctions_by_retainer.items():
-        value = "\n".join(
-            f"[{auction['real_name']}]({auction['link']})" for auction in details
-        )
-        fields.append({"name": f"**{retainer}**", "value": value, "inline": True})
-
-    embed = create_embed(title, description, fields)
-    send_to_discord(embed, webhook_url)
+        values = []
+        for auction in details:
+            if check_auction_is_new(auction):
+                values.append(f"[{auction['real_name']}]({auction['link']}) — Mine: {auction['my_ppu']}, {auction['undercut_retainer']}: {auction['ppu']}")
+        value = "\n".join(values)
+        if values:
```

Comment:
This is adjusted to not assume there will be content to post, because we now iterate over each entry first and there may be no auctions left after checking.